tt_metal/impl/device/device.cpp

```diff
@@ -226,9 +247,14 @@ void Device::initialize_and_launch_firmware() {
     }
 
     // Load erisc app base FW to eth cores
-    for (const auto &eth_core : this->get_active_ethernet_cores()) {
-        CoreCoord phys_eth_core = this->ethernet_core_from_logical_core(eth_core);
+    auto active_eth_cores = this->get_active_ethernet_cores();
```

Comment:
can we do:
```
for (eth_core: this->get_active_ethernet_cores())
```
for all erisc.cc FW and
```
for (eth_core: this->get_inactive_ethernet_cores())
```
for all idle eth cores.

Essentially, `get_inactive_ethernet_cores()` should return you the logical coords for `not_done_idle_eth_cores`